addon/components/container-query.js

```diff
@@ -49,7 +49,15 @@ export default class ContainerQueryComponent extends Component {
     const prefix = this.dataAttributePrefix;
 
     for (const [featureName, meetsFeature] of Object.entries(this.queryResults)) {
-      const attributeName = `data-${prefix}-${featureName}`;
+      let attributeName;
```

Comment:
An alternative approach to defining `attributeName` is:

```javascript
const attributeName = ['data', prefix, featureName]
  .filter(string => string)
  .join('-');
```

This is equivalent to the code that I committed in this PR, all except for when `featureName === ''` (an empty string is a valid key for POJOs).

Although this edge case (edge-edge case?) is unlikely to occur in practice, I decided to go with the `if-else` approach so that, if `featureName === ''`, then we will create a 1-1 mapping of data attributes. (The `filter-join` approach collapses two cases into 1.) The user will see the data attribute `data-`.